packages/repo-metadata-lint/src/validate.ts

```diff
@@ -0,0 +1,96 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import {URL} from 'url';
+
+export interface ValidationResult {
+  status: 'success' | 'error';
+  errors: Array<string>;
+}
+
+const LIBRARY_TYPES = [
+  'GAPIC_AUTO',
+  'GAPIC_MANUAL',
+  'AGENT',
+  'MISC',
```

Comment:
Library type spec allows for type "OTHER" as opposed to "MISC". I'm open to change but would likely require some refactoring of DRIFT workers

Reply:
I'm happy with `OTHER` rather than `MISC`, as long as there's something to use as a catch all.

Reply:
switched to `OTHER`.